src/ima.js

```diff
@@ -1099,7 +1099,7 @@ class Ima extends BasePlugin implements IMiddlewareProvider, IAdsControllerProvi
    */
   _maybeForceExitFullScreen(): void {
     const isIOS = () => this.player.env.os.name === 'iOS';
-    if (isIOS() && !this._adsManager.isCustomPlaybackUsed() && this.player.isFullscreen()) {
+    if (isIOS() && !this._adsManager.isCustomPlaybackUsed() && this.player.isFullscreen(true)) {
```

Comment:
see note [here](https://github.com/kaltura/playkit-js/pull/353#discussion_r270634302), just check against player config if the flag for inBrowserFullscreen is false